data/json/recipes/recipe_food.json

```diff
@@ -7230,5 +7230,28 @@
         [ "blackberries", 1 ]
       ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "jar_pickles_pickled",
```

Comment:
As far as I remember the recipes that produce fermenting products that need a day to "ferment" include pickles, pickled eggs, sauerkraut. Each of them IR needs time from being made to being edible as a final product. That time may be short but it's a part of the process and it is simulated here. Game time for conversion isn't too long either. Please don't change that, it would be a step back in simulating RL conditions.

Reply:
No, this is fine -- but if that's the case, we need to do the same for pickled fish, pickled meat, and pickled human as well.